pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -194,6 +196,16 @@ func (a *AgentClusterInstall) Generate(dependencies asset.Parents) error {
 			icOverrides.Capabilities = installConfig.Config.Capabilities
 			icOverridden = true
 		}
+
+		if installConfig.Config.AdditionalTrustBundle != "" {
+			// If AdditionalTrustBundle is set but mirror registries are not configured
+			// the bundle will not be included by assisted in the installed image.
+			// In this case add it to the config overrides to be included.
+			if len(installConfig.Config.DeprecatedImageContentSources) == 0 && len(installConfig.Config.ImageDigestSources) == 0 {
```

Comment:
```suggestion
			if len(installConfig.Config.DeprecatedImageContentSources) == 0 && len(installConfig.Config.ImageDigestSources) == 0 {
```

Just always mirror the AdditionalTrustBundle from the install-config to the assisted overrides, unconditionally, for the reasons I mentioned above